lib/main.coffee

```diff
@@ -3,57 +3,77 @@ hyperclickProvider = require './hyperclickProvider'
 
 module.exports =
   config:
-    caseInsensitiveCompletion:
-      type: 'boolean'
-      default: true
-      title: 'Case Insensitive Completion'
-      description: 'The completion is by default case insensitive.'
     showDescriptions:
       type: 'boolean'
       default: true
+      order: 1
       title: 'Show descriptions'
       description: 'Show doc strings from functions, classes, etc.'
-    outputProviderErrors:
-      type: 'boolean'
-      default: false
-      title: 'Output Provider Errors'
-      description: 'Select if you would like to see the provider errors when they happen. By default they are hidden. Note that critical errors are always shown.'
-    outputDebug:
-      type: 'boolean'
-      default: false
-      title: 'Output Debug Logs'
-      description: 'Select if you would like to see debug information in developer tools logs. May slow down your editor.'
     useSnippets:
       type: 'string'
       default: 'none'
+      order: 2
       enum: ['none', 'all', 'required']
       title: 'Autocomplete Function Parameters'
-      description: 'Automatically complete function arguments after typing left parenthesis character. Use completion key to jump between arguments.'
-    pythonPath:
+      description: '''Automatically complete function arguments after typing
+      left parenthesis character. Use completion key to jump between
+      arguments.'''
+    pythonPaths:
       type: 'string'
       default: ''
-      title: 'Path to python directory'
-      description: 'Optional. Set it if default values are not working for you or you want to use specific python version. For example: `/usr/local/Cellar/python/2.7.3/bin` or `E:\\Python2.7`'
-    pythonExecutable:
-      type: 'string'
-      default: 'python'
-      enum: ['python', 'python2', 'python3']
-      title: 'Python executable name'
-      description: 'Set it if default values are not working for you or you want to use specific python version.'
+      order: 3
+      title: 'Python executable paths'
+      description: '''Optional semicolon separated list of paths to python
+      executables (including executable names), where the first one will take
+      higher priority over the last one. By default autocomplete-python will
+      automatically look for venvs inside of your project and try to use them
+      as well as try to find global python executable.
+      Use $PROJECT variable for project specific paths to point on executables
+      in virtual environments.
+      For example: `$PROJECT/env/bin/python3;/usr/bin/python`.
+      Such config will fall back on `/usr/bin/python` for projects without venv.
```

Comment:
Consider enclose "venv" in backticks for consistency with the previous sentence:

> ... for projects without `venv`.

(See my previous comment.)